workday-application/pom.xml

```diff
@@ -262,7 +294,8 @@
 <!--&lt;!&ndash;                            <goal>copy-resources</goal>&ndash;&gt;-->
 <!--&lt;!&ndash;                        </goals>&ndash;&gt;-->
 <!--&lt;!&ndash;                        <configuration>&ndash;&gt;-->
-<!--&lt;!&ndash;                            <outputDirectory>${project.build.outputDirectory}/static/images</outputDirectory>&ndash;&gt;-->
+<!--&lt;!&ndash;                            <outputDirectory>${project.build.outputDirectory}/static/images
+                            </outputDirectory>&ndash;&gt;-->
```

Comment:
Let's remove this entire commented out plugin (leftover from the vite migration: https://github.com/flock-community/flock-eco-workday/pull/428/changes#diff-0d1e4f48d1ec6c1a263443d2f9ae4643e9766692df3e4f377e3c45a77366b517L254)